cuebot/src/main/resources/conf/ddl/postgres/migrations/V18_Add_New_Indexes

```diff
@@ -0,0 +1,29 @@
+
+--Performance issue, Created new index on column int_gpus_min
+
+
+CREATE INDEX IF NOT EXISTS i_layer_int_gpu_mem_min
+    ON public.layer USING btree
+    (int_gpus_min ASC NULLS LAST)
+    TABLESPACE pg_default;
+    
+
+CREATE INDEX IF NOT EXISTS i_layer_int_gpu_mem_min_1
+    ON public.layer USING btree
+    (int_gpu_min ASC NULLS LAST)
+    TABLESPACE pg_default;
+
+
+create index concurrently i_layer_int_cores_max on layer(int_cores_max);
```

Comment:
nit: use consistent capitalization within the file -- preferred `CREATE INDEX CONCURRENTLY ... ON ...`